src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        int ballsToCreate = 3;
+        int numberRange = 100;
```

Comment:
Similarly, `numberRange` should be defined as a constant to avoid using magic numbers. Define it as `private static final int NUMBER_RANGE = 100;`.